eclcli/bare/bareclient/base.py

```diff
@@ -22,16 +22,19 @@
 import abc
 import contextlib
 import hashlib
-import inspect
 import os
 import threading
 import json
```

Comment:
I think that it is necessary to describe like other processing.
```suggestion
try:
    import json
except ImportError:
    import simplejson as json
```